app/component/ItineraryPage.js

```diff
@@ -87,7 +79,21 @@ import ItineraryShape from '../prop-types/ItineraryShape';
 import ErrorShape from '../prop-types/ErrorShape';
 import RoutingErrorShape from '../prop-types/RoutingErrorShape';
 
-const POINT_FOCUS_ZOOM = 16; // used when focusing to a point
+const streetHashes = ['walk', 'bike', 'bikeAndVehicle', 'car', 'parkAndRide'];
```

Comment:
Not sure if this could belong to path.js or something and maybe again use some enum like thing so we can compare against it when we check what the hash is.

Reply:
Fixed.  Consistent  sharing of string constants is not used yet, but maybe it is good time to get started.